sdk/typescript/src/dwallet-mpc/dkg.ts

```diff
@@ -40,13 +40,15 @@ export interface DWallet {
 	session_id: string;
 	dwallet_cap_id: string;
 	output: number[];
+	dwallet_mpc_network_key_version: number;
```

Comment:
```suggestion
	network_key_version: number;
```

Reply:
There is another key named as "network key" within the sui code